src/fal/planner/tasks.py

```diff
@@ -89,9 +89,16 @@ def _run_script(script: FalScript) -> Dict[str, Any]:
     # DBT seems to be dealing with only UTC times
     # so we'll follow that convention.
     started_at = datetime.now(tz=timezone.utc)
+
     try:
-        with _modify_path(script.faldbt):
+        from fal_runtime.client import runtime_run
+    except:
+        def runtime_run(script: FalScript):
```

Comment:
Not super sure a user always wants to use the runtime if the package is installed. Should we check a fal_project.yml property? a `~/.dbt/profiles.yml`? Somwhere?